content/repositories/viewing-activity-and-data-for-your-repository/viewing-traffic-to-a-repository.md

```diff
@@ -18,6 +18,16 @@ You can navigate to referring sites, excluding search engines and {% data variab
 
 Referring sites and popular content are ordered by views and unique visitors. Full clones and visitor information update hourly, while referring sites and popular content sections update daily. All data in the traffic graph uses the UTC+0 timezone, regardless of your location.
 
+The insights tab within your repository displays traffic for your repo, not your GitHub pages site itself.
```

Comment:
Most repositories do not have GitHub Pages sites, so let's phrase this in a way that doesn't assume they do. I'm also rearranging bit so that both sentences about Pages are together.

```suggestion
The traffic graph displays traffic for your repository, not any {% data variables.product.prodname_pages %} sites that may be published from the repository. If you want to track traffic to a {% data variables.product.prodname_pages %} site, you can configure a third-party analytics service.
```